### DISCUSSION TITLE
Can I modify the labels (eg title-block-modified)?

### DISCUSSION THREAD

**Discussion by giabaio** [quarto-dev/quarto-cli#3928] [Q&A] [answered]
I think I understand how you can set your global language to something different than the default English, for which, I think, the labels that map onto key variables are set under  `src/resources/language`. But what if I want my website to speak English, while instead of "Modified" saying "Last updated" whenever `date-modified` is set? Do I need to create a custom `_language-**.yml` file and point my `_quarto.yml` to use that? 

This may be a very simple issue, so apologies if trivial!
Thanks

**Comment by giabaio** [comment, marked as answer]
Forget about it --- seems like all it takes it to add the key and label onto the `_quarto.yml` file, something like
```
...
title-block-modified: "Last updated"
```

**Reply by cderv** [reply]
for reference the doc is here
[quarto.org/docs/authoring/language.html#alternate-language](https://quarto.org/docs/authoring/language.html#alternate-language)

**Comment by cderv** [comment]
Yes you can modified a single element directly in YAML - for reference the doc is here
https://quarto.org/docs/authoring/language.html#alternate-language

**Comment by giabaio** [comment]
Awesome! Thank you!
G

---

Gianluca Baio
Professor of Statistics and Health Economics

Department of Statistical Science | University College London
1-19 Torrington Place, London, WC1E 6BT, UK
Twitter: @stats_ucl<https://twitter.com/stats_UCL> | Instagram: @stats_UCL<https://instagram.com/ucl.stats/>

Telephone: +44(0)20 7679 1248 (internal: 41248)
Website: http://www.homepages.ucl.ac.uk/~ucakgba
Blog: https://gianluca.statistica.it/blog/
Book a meeting with me: https://book.morgen.so/gianlucabaio/book-me

***@***.***>   [https://www.svgrepo.com/show/157815/twitter.svg] <https://twitter.com/gianlubaio>    [https://www.svgrepo.com/show/138936/linkedin.svg] <https://www.linkedin.com/in/gianluca-baio-b893879/>    [https://www.svgrepo.com/show/341847/github.svg] <https://github.com/giabaio>    [https://www.geol.umd.edu/styles/academicons-192/svg/arxiv.svg] <https://arxiv.org/a/baio_g_1.html>    [https://www.geol.umd.edu/styles/academicons-192/svg/orcid.svg] <https://orcid.org/0000-0003-4314-2570>    [https://www.svgrepo.com/show/40973/soundcloud.svg] <https://soundcloud.com/uclsound/sets/sample-space>

________________________________

For Head of Department correspondence, please use ***@***.******@***.***>

[https://s1g.s3.amazonaws.com/3f32ef1962be41fb8943ee58d28cf3aa.png]

This email may reach you outside of your working hours. If that’s the case, please do not feel pressure to reply immediately.


On January 11, 2023 at 13:48 GMT, Christophe Dervieux ***@***.******@***.***>> wrote:


⚠ Caution: External sender


Yes you can modified a single element directly in YAML - for reference the doc is here
https://quarto.org/docs/authoring/language.html#alternate-language<https://eur01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fquarto.org%2Fdocs%2Fauthoring%2Flanguage.html%23alternate-language&data=05%7C01%7Cg.baio%40ucl.ac.uk%7Cf0accd07def0488323e008daf3da7448%7C1faf88fea9984c5b93c9210a11d9a5c2%7C0%7C0%7C638090416833355772%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=GnzxC3mzHTlqAaiuGx3iKHo91IjNVbWHCAzjSUoQwX4%3D&reserved=0>

—
Reply to this email directly, view it on GitHub<https://github.com/quarto-dev/quarto-cli/discussions/3928#discussioncomment-4656551>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/ACCRVSM5GMK652V4A6EXPBTWR22Y7ANCNFSM6AAAAAATYBOZ2E>.
You are receiving this because you authored the thread.Message ID: ***@***.***>

[https://bolt.im/t/?_N3hIjFheJTz9Sd9QSpLX54lvqBmtur7WWWXtQxw_4B3ptzhJG0Zp_pyoNbpFq4iEzfg9axQBFoU3aTPKoIpSw]
